appveyor.yml

```diff
@@ -1,6 +1,9 @@
 image: Visual Studio 2017
 build_script:
-  - build.cmd Test
-test: off
+  - build.cmd
```

Comment:
If we just want to build, without running tests, we can use `build.cmd Build`. Without argument it will run the default target, which is currently `Test` (but we should not rely on the default target here).

Reply:
noted, will change the pr to only do the tests once (as was originally envisaged)